addons/payment_mollie_official/models/payment_acquirer.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+from mollie.api.client import Client
+from odoo import models, fields, api, _
+from odoo.http import request
+
+import base64
+import requests
+import logging
+_logger = logging.getLogger(__name__)
+
+
+class PaymentAcquirer(models.Model):
+    _inherit = 'payment.acquirer'
+
+    _mollie_client = Client()
+
+    provider = fields.Selection(selection_add=[('mollie', 'Mollie')])
+    mollie_api_key_test = fields.Char(
```

Comment:
not a fan, but since mollie provides both it shouldn't block any customer.

Reply:
There's no other way I guess. We need to be able to fill in the test API key for test transactions after all.